doc/news/changes/incompatibilities/20200324MartinKronbichler

```diff
@@ -0,0 +1,5 @@
+Deprecated: The macro DEAL_II_COMPILER_VECTORIZATION_LEVEL (using values 1, 2,
```

Comment:
can you update this (we don't deprecate after all).

Reply:
Right, let us move this to minor instead.